package.json

```diff
@@ -239,6 +239,7 @@
     "simple-git": "3.24.0",
     "slugify": "1.6.6",
     "source-map-support": "0.5.21",
+    "tar": "6.2.1",
```

Comment:
I have moved the `tar` dependency from `devDependencies` to `dependencies` as `PkgServer`s serve tarballs that need to be extracted. I hope this is okay. (Re)using this dependency seemed the most straight-forward as it was already included as a development dependency and therefore likely has gone through some scrutiny.